### ISSUE TITLE
The repacked gme file does not contain the changed files

### ISSUE THREAD

**Issue by termoplastikon** [adventurebrew/magos#5]
I want to change the VGA and TEXT files located in the GME file. To do this:

1. I extract the contents of the GME file using the following command:

> magos SIMON.GME -g simon1 -e ABC

2. I paste my modified TEXT and VGA files into the ABC directory.
3. I create a new GME file using the command: 

> magos SIMON.GME -g simon1 -e ABC -r

Unfortunately, the GME file created this way still contains the original TEXT files instead of the ones I placed in the ABC directory.

**Comment by BLooperZ** [comment]
Hi, thank you for reporting
I think text files would be overwritten by the text in the strings.txt file which is always created, so it is possible that only VGA files were modified.
can you please verify by extracting the created file and comparing with the files extracted from the original?

**Comment by termoplastikon** [comment]
You are right. The content of string.txt overwrites the files from the ABC directory and if you remove the string.txt file before repacking files you get an error.

**Comment by BLooperZ** [comment]
I would like to suggest trying to extract the texts from these files to new strings.txt

afyer you have the directory with the extracted files with some files overwritten
copy non-GME files from the original game to this directory as well and run
magos -g simon1 -m ABC

you should get new strimgs.txt with the texts from new replacent files, use it for rebuildimg the origonal ge directory as you did
